cfglib.py

```diff
@@ -211,7 +211,17 @@
     cfg.BoolOpt('block_migration', default=False,
                 help='live-migration without shared_storage'),
     cfg.StrOpt('host_eph_drv', default='-',
-               help='host ephemeral drive')
+               help='host ephemeral drive'),
+    cfg.StrOpt('connection', default='mysql+mysqlconnector',
+               help='driver for db connection'),
+    cfg.StrOpt('host', default='',
```

Comment:
It is definitely good idea, but there are a lot of name changes we should do in the whole project (OpenStack custom db settings, general db settings) and do not break everything. In this case, I would suggest to leave everything as it is, create a 'nice to have' issue with this renaming and do it as a separate refactoring task someday. What do you think?